cmd/kubeadm/app/phases/certs/certs_test.go

```diff
@@ -147,6 +192,14 @@ func TestGetAltNames(t *testing.T) {
 			expectedIPs:      []string{"192.168.200.1", "10.96.0.1"},
 			expectedDNSNames: []string{"my-node", "kubernetes", "kubernetes.default", "kubernetes.default.svc", "kubernetes.default.svc.cluster.external", "foo", "bar.baz"},
 		},
+		{
+			cfgaltnames:      []string{"foo", "2001:db8::1", "bar.baz"},
+			hostname:         "my-node",
+			dnsdomain:        "cluster.external",
+			servicecidr:      "2001:ca1::1/48",
+			expectedIPs:      []string{"2001:db8::1", "2001:ca1::1"},
+			expectedDNSNames: []string{"my-node", "kubernetes", "kubernetes.default", "kubernetes.default.svc", "kubernetes.default.svc.cluster.external", "foo", "bar.baz"},
```

Comment:
Same here, even though the Errorf() shows the expected values and not true/false, both test cases have the same expected DNS names, so it's hard to tell the failing test.